packages/frontend/src/components/host-chain-risks-warning.tsx

```diff
@@ -0,0 +1,40 @@
+import { type ScalingProjectDisplay } from '@l2beat/config'
+import Image from 'next/image'
+import Link from 'next/link'
+
+export type HostChainRisksWarningProps = {
+  hostChain: ScalingProjectDisplay
+  riskCount?: number
+}
+
+export function HostChainRisksWarning({
```

Comment:
![image](https://github.com/user-attachments/assets/9da0644e-84f0-4936-be85-e82ff26d271e)
It's not centered

Reply:
https://github.com/l2beat/l2beat/pull/5318/commits/6932bfaa417b2a9306a5a3aa8f7749d1fb59eb22

Reply:
https://github.com/l2beat/l2beat/pull/5318/commits/275a18de09e65285303be0988320b3a822a860b5